docs/form-elements/autocomplete.md

```diff
@@ -14,6 +14,7 @@ Allows the user to select a single option from a list of options which can be fi
 | `defaultValue` | Yes      | `string`  |                  | The identifier of an option to be selected when the form is opened.                      |
 | `required`     | Yes      | `boolean` | `false`          | Determine if this element requires an option to be selected (`true`) or not (`false`).   |
 | `readOnly`     | Yes      | `boolean` | `false`          | Determine if this selected option be changed by the user (`false`) or not (`true`).      |
+| `searchUrl`    | No       | `string`  |                  | The URL from which options will be requested periodically if the option type is `SEARCH`. The current input of the Autocomplete element will be passed to this URL as the query string property `value`. |
```

Comment:
> The URL from which options will be requested periodically if the option type is `SEARCH`

Took me a couple of reads to get this sentence. 

perhaps something like "The URL to send the value of the autocomplete element to. The value will be sent on user input (debounced by xxx ms) as the query string property `value`. It should return the relevant options in (some format)